src/main/resources/com/google/api/codegen/csharp/gapic_client.snip

```diff
@@ -457,6 +460,9 @@
         public partial class {@method.listMethod.requestTypeName} : IPageRequest { }
         public partial class {@method.listMethod.responseTypeName} : IPageResponse<{@method.listMethod.resourceTypeName}>
         {
+            /// <summary>
+            /// Returns an enumerator that iterates through the resources in this response.
+            /// </summary>
             public IEnumerator<{@method.listMethod.resourceTypeName}> GetEnumerator() => {@method.listMethod.resourcesFieldGetFunction}.GetEnumerator();
             IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
```

Comment:
We should have something on here as well - just `/// <inheritdoc/>` would be fine, I think.